libcxx/include/__memory_resource/monotonic_buffer_resource.h

```diff
@@ -27,8 +27,8 @@ namespace pmr {
 // [mem.res.monotonic.buffer]
 
 class _LIBCPP_AVAILABILITY_PMR _LIBCPP_EXPORTED_FROM_ABI monotonic_buffer_resource : public memory_resource {
-  static const size_t __default_buffer_capacity  = 1024;
-  static const size_t __default_buffer_alignment = 16;
```

Comment:
Was `__default_buffer_alignment` unused?

Reply:
It seems that `__default_buffer_alignment` has never been used since intiallly added (243da90ea5357c1ca324f714ea4813dc9029af27, https://reviews.llvm.org/D89057).